drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-kogito-client/src/main/java/org/drools/workbench/screens/scenariosimulation/kogito/client/dmn/ClientDMNType.java

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright 2020 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.drools.workbench.screens.scenariosimulation.kogito.client.dmn;
+
+import java.util.Map;
+
+import org.drools.workbench.screens.scenariosimulation.kogito.client.dmn.feel.BuiltInType;
+
+public class ClientDMNType {
+
+    private String namespace;
+    private String name;
+    private String id;
+    private boolean collection;
+    private boolean composite;
```

Comment:
This can be removed and replaced by `fields.size() > 0`. Wdyt?

Reply:
@danielezonca At first glance it was a good idea but, applying it, we will lost the possibility to manage this particular case:
`                if (type == BuiltInType.CONTEXT) {
                    feelPrimitiveType = new ClientDMNType(URI_FEEL, name, null, false, true, Collections.emptyMap(), type);
                }`